docs/entity/adapter.md

```diff
@@ -87,6 +87,7 @@ state if no changes were made.
 - `addMany`: Add multiple entities to the collection
 - ~~`addAll`~~: (Deprecated and renamed to `setAll`). ~~Replace current collection with provided collection~~
 - `setAll`: Replace current collection with provided collection
+- `setOne`: Add or Replace on entity in the collection
```

Comment:
Could you move these changes to https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/guide/entity/adapter.md so they become visible on the website 🙂?
These docs are for older versions of NgRx

Reply:
Done